presidio-analyzer/presidio_analyzer/predefined_recognizers/in_aadhaar_recognizer.py

```diff
@@ -0,0 +1,77 @@
+from stdnum import verhoeff
+from presidio_analyzer import Pattern, PatternRecognizer
+from typing import Optional, List, Tuple
+
+
+class InAadhaarRecognizer(PatternRecognizer):
+    """
+    Recognizes Indian UIDAI Person Identification Number ("AADHAAR").
+
+    Reference: https://en.wikipedia.org/wiki/Aadhaar
+    A 12 digit unique number that is issued to each individual by Government of India
+    :param patterns: List of patterns to be used by this recognizer
+    :param context: List of context words to increase confidence in detection
+    :param supported_language: Language this recognizer supports
+    :param supported_entity: The entity this recognizer can detect
+    :param replacement_pairs: List of tuples with potential replacement values
+    for different strings to be used during pattern matching.
+    This can allow a greater variety in input, for example by removing dashes or spaces.
+    """
+
+    PATTERNS = [
+        Pattern(
+            "AADHAAR (Low)",
```

Comment:
As this is a very weak detector, I would suggest changing the pattern name to AADHAAR (very week) and the default score to 0.01

Reply:
thanks for the suggestion. implemented as desired.